src/com/goide/ui/ProjectTutorialNotification.java

```diff
@@ -37,14 +35,12 @@ public void initComponent() {
     boolean shownAlready;
     //noinspection SynchronizationOnLocalVariableOrMethodParameter
     synchronized (propertiesComponent) {
-      String storedVersion = propertiesComponent.getValue(GO_PROJECT_TUTORIAL_NOTIFICATION_SHOWN, "0.0");
-      shownAlready = VersionComparatorUtil.compare(storedVersion, PLUGIN_VERSION) == 0;
-      propertiesComponent.setValue(GO_PROJECT_TUTORIAL_NOTIFICATION_SHOWN, PLUGIN_VERSION);
+      String isShown = propertiesComponent.getValue(GO_PROJECT_TUTORIAL_NOTIFICATION_SHOWN, "n");
```

Comment:
Done, I'm sorry

On Thu, Apr 14, 2016 at 10:57 PM Alexander Zolotov <notifications@github.com>
wrote:

> In src/com/goide/ui/ProjectTutorialNotification.java
> <https://github.com/go-lang-plugin-org/go-lang-idea-plugin/pull/2464#discussion_r59798943>
> :
>
> > @@ -37,14 +35,12 @@ public void initComponent() {
> >      boolean shownAlready;
> >      //noinspection SynchronizationOnLocalVariableOrMethodParameter
> >      synchronized (propertiesComponent) {
> > -      String storedVersion = propertiesComponent.getValue(GO_PROJECT_TUTORIAL_NOTIFICATION_SHOWN, "0.0");
> > -      shownAlready = VersionComparatorUtil.compare(storedVersion, PLUGIN_VERSION) == 0;
> > -      propertiesComponent.setValue(GO_PROJECT_TUTORIAL_NOTIFICATION_SHOWN, PLUGIN_VERSION);
> > +      String isShown = propertiesComponent.getValue(GO_PROJECT_TUTORIAL_NOTIFICATION_SHOWN, "n");
>
> propertiesComponent.getBoolean
> propertiesComponent.setValue(key, boolean)
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/go-lang-plugin-org/go-lang-idea-plugin/pull/2464/files/0a3ecb30282424d6a038e2ffafbb481548f277f5#r59798943>
>